lib/src/utils/confirmations/transaction_sender.dart

```diff
@@ -1,8 +1,7 @@
 /// SPDX-License-Identifier: AGPL-3.0-or-later
 import 'dart:async';
 
-import 'package:archethic_lib_dart/src/model/transaction.dart';
-import 'package:archethic_lib_dart/src/utils/confirmations/transaction_remote.dart';
+import 'package:archethic_lib_dart/archethic_lib_dart.dart';
```

Comment:
Quick question: I usually tend to import objects from `src` folder rather than `archethic_lib_dart.dart` file as a whole. What's the best practice?

Reply:
I'm not sure about which one is the best '-_-.

But I agree with your suggestion. That change was not made on purpose...

I fix this.